### DISCUSSION TITLE
Room DB 시각화 보기

### DISCUSSION THREAD

**Discussion by dlwlstks96** [euphony-io/check-in-listener#27] [General]
DB 관련하여 테스트를 시도해보다 좋은 방법을 찾은 것 같아 혹시 도움이 될까하여 공유합니다!!
Room DB에 저장되어 있는 데이터를 브라우저에서 쉽게 눈으로 확인 및 수정할 수 있습니다:)
[Room DB 시각화-깃허브](https://github.com/amitshekhariitbhu/Android-Debug-Database)
[Room DB 시각화-관련블로그](https://sjhp.tistory.com/3)
- 결과
![결과2](https://user-images.githubusercontent.com/72801465/135442657-5b101cb4-a680-46d8-b4bd-bcd6407c2905.JPG)

0. 참고사항
- Android 휴대폰과 PC는 동일한 네트워크(Wifi 또는 LAN)에 연결되어 있어야 합니다.
1. build.gradle에 아래 코드 추가
```
debugImplementation 'com.amitshekhar.android:debug-db:1.0.6'
```

2. Logcat 창에 뜨는 아래 코드 확인
```
D/DebugDB: Open http://XXX.XXX.X.XXX:8080 in your browser
```
3. USB 혹은 애뮬레이터를 사용하는 경우엔 아래 코드를 터미널에 작성
```
adb forward tcp:8080 tcp:8080
```

4. 그럼에도 위 2번 순서의 링크 접속이 안될시 주소창에 localhost:8080 입력하여 접속 시도

**Comment by Jsueeee** [comment]
와 Room으로 이렇게 볼 수 있는지 처음 알았네요 멋있습니다 👍👍👍👍👍

**Comment by sujin9** [comment]
정말 편리한 것 같아요!!! 감사합니다 😊👍
